concent_api/concent_api/middleware.py

```diff
@@ -39,3 +43,33 @@ def __call__(self, request):
         response = self.get_response(request)
         response['Concent-Version'] = self._concent_version
         return response
+
+
+class HandleServerErrorMiddleware(object):
+    """
+    Used to catch all unhandled exceptions and return JSON response containing error information.
+    """
+    def __init__(self, get_response):
+        self.get_response = get_response
+        self.sid = None
+
+    def __call__(self, request):
+        self.sid = transaction.savepoint()
+        response = self.get_response(request)
+        transaction.savepoint_commit(self.sid)
+        return response
+
+    def process_exception(self, request, exception):  # pylint: disable=unused-argument
+        transaction.savepoint_rollback(self.sid)
+        logging.logger.exception(exception)
+        return self._build_json_response(exception)
+
+    def _build_json_response(self, exception):  # pylint: disable=no-self-use
+        message = {"error": str(exception)}
```

Comment:
Please call the field `error_message` for consistency with #131.

Reply:
done